src/Microsoft.TestPlatform.ObjectModel/CodeCoverage/TestSDKAutoGeneratedCode.cs

```diff
@@ -0,0 +1,11 @@
+﻿// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT license. See LICENSE file in the project root for full license information.
+
+namespace Microsoft.VisualStudio.TestPlatform
+{
+    using System;
+
+    public class TestSDKAutoGeneratedCode : Attribute
```

Comment:
Nit: Doc.

Reply:
Can we have general name similar to earlier commits which can solve https://github.com/Microsoft/vstest/issues/1548 .